app/views/users/edit.js.erb

```diff
@@ -0,0 +1,2 @@
+$('.profile-flash-container').html("<%= j render('shared/flash', type: 'error', content: flash[:error]) %>");
```

Comment:
I think i'm probably misunderstanding what's going on here, but doesn't this just render an error flash into the edit page? Doesn't rails have built in flash rendering?

Reply:
I don't think there's a built in flash renderer. Rails just provides a way for you to handle and manipulate the contents of the different types of flash messages. I believe it's up to you to decide how you want to display it.